nexp/clients/data.py

```diff
@@ -353,12 +353,20 @@ def candidates_for_facility(self, facility: Any) -> ModelIterator:
                   AND json_extract(c.fields, "$.unavailable") is null
                       {clause}
 
+            ), tracked_candidates AS (
+                SELECT substring_index(tracked_candidate.value, ',', 1) AS name
+                FROM tracking t
+                     , json_each(t.fields, "$.candidates") tracked_candidate
+                JOIN facilities f
+                ON f.facility_name = t.facility
             )
             SELECT *
 
               FROM candidates c
 
               JOIN distinct_candidate_ids USING (id)
+              LEFT JOIN tracked_candidates USING (name)
```

Comment:
and the anti-join based on 
left joining tracked_candidates.name  and c.name

Reply:
I think we want to highlight new rows rather than remove old ones